### PR TITLE
Added ellipsoid values to socetlinescankeywords. Fixes #5362.

### PR DESCRIPTION
**Pull request by jessemapel**
Test results can be found under `/work/projects/isis/latest/m05362_5363/test_results` there is new test data for several socetlinescankeywords app tests. It can be found in `/work/projects/isis/latest/m05362_5363/ISIS3/isis/src/socet/apps/socetlinescankeywords/tsts`.

### CODE REVIEW

**Comment by jessemapel** [comment]
Redmine ticket [5362](https://isis.astrogeology.usgs.gov/fixit/issues/5362)

**Comment by jessemapel** [comment]
DO NOT SQUASH ON MERGE.  We want to preserve the commit that has the socetframesettings changes for future work.

File: isis/src/socet/apps/socetlinescankeywords/socetlinescankeywords.cpp [hunk 1]
```diff
@@ -694,6 +694,27 @@ void IsisMain() {
...
+    catch (IException &e) {
+      QString msg = "Failed to get target body radii from cube.";
+      throw IException(e, IException::Programmer, msg, _FILEINFO_);
+    }
+  }
+  double eccentricity = 1.0 - (targetRadii[2].meters() * targetRadii[2].meters())
+                            / (targetRadii[0].meters() * targetRadii[0].meters());
+  eccentricity = sqrt(eccentricity);
+  toStrm << "SEMI_MAJOR_AXIS  " << targetRadii[0].meters() <<"\n";
+  toStrm << "ECCENTRICITY     " << eccentricity <<"\n";
```
Comment (AgoinsUSGS) [inline isis/src/socet/apps/socetlinescankeywords/socetlinescankeywords.cpp]: Need some spacing after the last << on the above 2 lines.

**Review by AgoinsUSGS — CHANGES_REQUESTED** [review]
Looks good, just need a some spacing added on a couple of lines and it's good to go.

**Comment by jessemapel** [comment]
Tests are still running, I'll put a note on here when they are finished. Previous tests of app and cat tests passed.

**Review by AgoinsUSGS — APPROVED** [review]

**Comment by celbal** [comment]
Did the testing finish and was it clean?

**Comment by jessemapel** [comment]
Testing finished and was clean.

**Comment by celbal** [comment]
So can it be merged?

**Pull request merged**